test/samlp.logout.session_store.tests.js

```diff
@@ -44,6 +45,14 @@ describe('samlp logout with Session Participants - Session Provider', function (
   var sessions = [], returnError;
   var samlIdPIssuer = 'urn:fixture-test';
   
+  var frozenTime;
+  before(() => {
+    frozenTime = new Date(Date.now());
+    timekeeper.freeze(frozenTime)
```

Comment:
I think you can pass in a `number` directly, e.g. `timekeeper.freeze(Date.now())`. That avoids the `new Date().getTime()` calls below.